audit/projects/k8s-artifacts-prod/service-accounts/k8s-infra-gcr-auditor@k8s-artifacts-prod.iam.gserviceaccount.com/description.json

```diff
@@ -1,8 +0,0 @@
-{
-  "displayName": "k8s-infra container image auditor",
-  "email": "k8s-infra-gcr-auditor@k8s-artifacts-prod.iam.gserviceaccount.com",
```

Comment:
this was NOT an expected outcome of https://github.com/kubernetes/k8s.io/issues/1642

I am going to need to undo/resolve this, I meant to delete the service
account related to vulndash and must have copy-pasted the wrong thing 😞 

This isn't world-breaking because IIRC we have silenced the auditor's alerts, but I'm glad we caught this